kivymd/uix/dialog/dialog.py

```diff
@@ -563,7 +563,8 @@ def show_confirmation_dialog(self, *args):
         :align: center
 
     :attr:`content_cls` is an :class:`~kivy.properties.ObjectProperty`
-    and defaults to `'None'`.
+    and defaults to `'None'`. This attribute is only available when
```

Comment:
The description of the property should be at the beginning of the documentation line:

```python
    property = ObjectProperty()
    """
    Description.

    :attr:`property` is an :class:`~kivy.properties. ObjectProperty `
    and defaults to `None`.
    """
```